lib/services/ibm_devops.rb

```diff
@@ -1,8 +1,9 @@
 class Service::IBMDevOpsServices < Service::HttpPost
-  string :ibm_id
+  title 'IBM Bluemix DevOps Services'
+  string :ibm_login
```

Comment:
Also will the service configuration display both the previous ibm_id and new ibm_login fields when this is deployed?